### ISSUE TITLE
Will-Change Transform Problem

### ISSUE THREAD

**Issue by 96LawDawg** [ArnoldSmith86/virtualtabletop#697]
#400 created a significant problem for users of Edge and Chrome.  The default deck uses "black" as a border color.  Edge and Chrome only recognize hexadecimal values.  Removing "will-change:transform" is now causing significant lag when dragging cards from the default deck around the board.

**Comment by bjalder26** [comment]
I tried changing the border in a deck, and it didn't resolve the issue, so I think I was wrong to associate those two issues.  I'm afraid we might need to choose between the problems will-change causes, and the ones it resolves.  I'm noticing that dragging a deck of cards by the pile handle also causes lag.

**Comment by 96LawDawg** [comment]
I have no idea what the problems were before.  I read #400 and didn't understand it nor did I notice any problems before (although they may have been there; I don't know).  What I do know is that Edge is now nearly useless as a browser with VTT.  I don't think that is a situation we want to be in.  Chrome also has lag, it is just not as bad.

**Comment by bjalder26** [comment]
Yeah, it is really bad in Edge.  The issues with Apples-to-Apples and the rendering issue when zooming in came from `will-change: transform`, but for Edge, getting rid of it is definitely worse.

Perhaps it would be possible to toggle `will-change: transform` on and off based on where cards are in a pile.  If that could be done, then I think the performance would be even better, and without the negative side-effects.

**Comment by bjalder26** [comment]
Try out PR #698.  So far it adds `will-change: transform` to piles.  It should at least partially address this issue, and it does not appear to cause the issues seen with Apples-to-Apples or the rendering issue when zooming.  I suspect it would also be helpful to add `will-change: transform` to cards that are not below other cards in a pile, but I'm not noticing any issues after adding it to piles, so some testing to identify issues would help.
